apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -15,6 +24,40 @@ interface Props {
   params: {id: string};
 }
 
+function ContactDetailsSlideOver({datasetId}: {datasetId?: string}) {
+  const organization = useCurrentPublisher.getState();
+  const t = useTranslations('ObjectDetails');
+
+  return (
+    <SlideOverDialog>
+      <SlideOverHeader />
+      <SlideOverContent>
+        <div className="flex-col gap-4 mt-4 flex">
+          <h2>{organization.name}</h2>
+          <span>
+            {organization.address?.streetAddress}
+            <br /> {organization.address?.postalCode}{' '}
+            {organization.address?.addressLocality},
+            <br /> {organization.address?.addressCountry}
+          </span>
+          {organization.url && (
+            <a href={organization.url}>{t('linkToWebsite')}</a>
+          )}{' '}
+          {datasetId && (
+            <a
+              href={`https://datasets.colonialcollections.nl/nl/datasets/${encodeURIComponent(
```

Comment:
Is there a way to not hard-code this, and to fetch the base URL ('https://datasets.colonialcollections.nl') from a config? Then we can also link to the dev server ('https://dev.datasets.colonialcollections.nl/').

Reply:
I have moved the URL to environment variables and added the variable to Vercel.